kpet/data.py

```diff
@@ -95,6 +96,143 @@ def normalize(arg):
         self.location_types = normalize(location_types)
 
 
+# TODO Rename to just "Pattern" once kpet-db is transitions to this type
+class GenericPattern(Object):  # pylint: disable=too-few-public-methods
+    """Generic target pattern"""
+
+    # Target field qualifiers
+    qualifiers = {"trees", "arches", "sets", "sources", "location_types"}
+
+    """An execution target pattern"""
+    def __init__(self, data):
+        """
+        Initialize an execution pattern.
+
+        Args:
+            positive:   True if the pattern is positive, False if negative.
```

Comment:
This is not on the `__init__` line nor used in the code

Reply:
Argh, an overlooked copy-paste. Thank you :)